src/generator.js

```diff
@@ -453,7 +453,7 @@ new Generator('python', path.join(__dirname, '..', 'python', 'docs'), {
       text = `${toSnakeCase(member.clazz.varName)}.${toSnakeCase(member.alias)}`;
   
     if (member.kind === 'event')
-      text = `${toSnakeCase(member.clazz.varName)}.on("${toSnakeCase(member.alias)}")`;
+      text = `${toSnakeCase(member.clazz.varName)}.on("${toSnakeCase(member.alias.toLowerCase())}")`;
```

Comment:
nit: drop toSnakeCase(

Reply:
I've changed it in: https://github.com/microsoft/playwright.dev/pull/106